src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -556,13 +553,23 @@ def send_webhooks(installation: RpcSentryAppInstallation, event: str, **kwargs:
     with SentryAppInteractionEvent(
         operation_type=SentryAppInteractionType.SEND_WEBHOOK,
         event_type=SentryAppEventType(event),
-    ).capture():
+    ).capture() as lifecycle:
         servicehook: ServiceHook
         try:
             servicehook = ServiceHook.objects.get(
                 organization_id=installation.organization_id, actor_id=installation.id
             )
         except ServiceHook.DoesNotExist:
+            lifecycle.add_extra("events", installation.sentry_app.events)
```

Comment:
nit: you can add `events` to the big dictionary below

Reply:
mypy :despairge. It was this or change type of add_extras func, I guess I go change type.